docs/user-guide/semdedup.rst

```diff
@@ -118,32 +118,32 @@ The semantic deduplication process is controlled by two key threshold parameters
 
     eps_to_extract: 0.01
 
-1. `eps_thresholds`: A list of similarity thresholds used to compute semantic matches. Each threshold represents a different level of strictness in determining duplicates.
+1. ``eps_thresholds``: A list of similarity thresholds used to compute semantic matches. Each threshold represents a different level of strictness in determining duplicates.
                      Lower values are more strict, requiring higher similarity for documents to be considered duplicates.
 
-2. `eps_to_extract`: The specific threshold used for the final extraction of deduplicated data.
-                     This value must be one of the thresholds listed in `eps_thresholds`.
+2. ``eps_to_extract``: The specific threshold used for the final extraction of deduplicated data.
+                     This value must be one of the thresholds listed in ``eps_thresholds``.
 
 This two-step approach offers several advantages:
-* Flexibility to compute matches at multiple thresholds without rerunning the entire process.
```

Comment:
I think `*` is the right format for unordered lists in `rst` files. I think we just need a blank line before the list begins. See [here](https://github.com/NVIDIA/NeMo-Curator/blob/main/docs/user-guide/download.rst).

Reply:
Updated, thanks!